README.md

```diff
@@ -1,23 +1,20 @@
 # pick_ik
 
-## Local development in devcontainer
+`pick_ik` is an inverse kinematics (IK) solver compatible with [MoveIt 2](https://github.com/ros-planning/moveit2).
 
-0. Install docker and add yourself to the docker group.
+The solver is a reimplementation of [`bio_ik`](https://github.com/PickNikRobotics/bio_ik), which combines
+* A local optimizer which solves inverse kinematics via gradient descent
```

Comment:
Please reference the specific solvers that were reimplemented. If it's something different, I would highlight that as well.

Reply:
Also, I would link the original repo https://github.com/TAMS-Group/bio_ik